app/(app)/home/about.tsx

```diff
@@ -1,34 +1,108 @@
- * -  consider reusing the image component that was used in the speakers list
- * -  clicking on the team member image should navigate to the [speaker] page
- */
-
-// TODO: Use data from mock/organizers.ts
+const { width } = Dimensions.get('window');
 
 const About = () => {
+  const { colors } = useTheme();
   const [signInModalVisible, setSignInModalVisible] = useState<boolean>(false);
 
   const showSignInModal = () => {
     setSignInModalVisible(true);
   };
+  const OrganizingIndividuals = OrganizingTeam.data.filter((item) => item.type === 'individual');
   return (
     <MainContainer preset="scroll">
       <Stack.Screen
         options={{
           headerRight: () => <HeaderRight handlePress={showSignInModal} />,
         }}
       />
-      <StyledText>about</StyledText>
+      <View style={styles.container}>
+        <Image
+          style={styles.image}
+          source="https://res.cloudinary.com/khariokitony/image/upload/v1690514752/Droidcon-organizers.png"
+          placeholder={blurhash}
+          contentFit="contain"
+        />
+
+        <View style={styles.content}>
+          <StyledText font="bold" size="xl" variant="primary">
+            About
+          </StyledText>
+
+          <Space size={16} />
+
+          <StyledText font="regular" size="md">
+            Droidcon is a global conference focused on the engineering of Android applications. Droidcon provides a
+            forum for developers to network with other developers, share techniques, announce apps and products, and to
+            learn and teach.
+          </StyledText>
+
+          <Space size={8} />
+
+          <StyledText font="regular" size="md">
+            This two-day developer focused gathering will be held in Nairobi Kenya on November 8th to 10th 2023 and will
+            be the largest of its kind in Africa.
+          </StyledText>
+
+          <Space size={8} />
+
+          <StyledText font="regular" size="md">
+            It will have workshops and codelabs focused on the building of Android applications and will give
+            participants an excellent chance to learn about the local Android development ecosystem, opportunities and
+            services as well as meet the engineers and companies who work on them.
+          </StyledText>
+
+          <Space size={30} />
+
+          <StyledText font="bold" size="xl" variant="primary">
+            Organizing Team
+          </StyledText>
+
+          <Space size={20} />
+
+          <FlatList
+            data={OrganizingIndividuals}
+            numColumns={3}
+            renderItem={({ item }: ListRenderItemInfo<OrganizingTeamMember>) => (
+              <View style={styles.item}>
+                <Image
+                  source={{ uri: item.photo }}
+                  style={[styles.avatar, { borderColor: colors.tint }]}
+                  contentFit="cover"
+                  placeholder={blurhash}
+                />
```

Comment:
I noticed in the design when you press an image it navigates you to a screen showing more info about team member. I guess we could use `[speaker].tsx` reusable. Make it show either speaker or team member

Reply:
done